demo/clusterctl-examples/sample-kubeadm-ctlplane-crio-ubuntu.rc

```diff
@@ -0,0 +1,260 @@
+# cluster and make sure the host/IP is outside the ranges
+# of any IP pool.
+#
+export CLUSTER_APIENDPOINT_HOST=192.168.75.98
+export CLUSTER_APIENDPOINT_PORT=6443
+
+#
+# IP range for the provisioning IP pool.
+# This should be the (internal) provisioning subnet.
+#
+export PROVISIONING_POOL_START=192.168.75.100
+export PROVISIONING_POOL_END=192.168.75.200
+# this is the CIDR prefix
+export PROVISIONING_POOL_PREFIX=24
+
+#
+# IP range for the baremetal IP pool.
+# This can be either the management or public subnet.
+#
+export BAREMETAL_POOL_GATEWAY=10.84.81.14
+export BAREMETAL_POOL_START=10.84.81.9
+export BAREMETAL_POOL_END=10.84.81.11
+# this is the CIDR prefix
+export BAREMETEAL_POOL_PREFIX=24
+
+#
+# Base image used to provision the workload cluster.
+#
+# NOTE: the IMAGE_CHECKSUM can either be the checksum value or the URL
+# to the checksum value.
+#
+export IMAGE_URL=http://media.suse.baremetal/ubuntu_22.04-efi.qcow2
+export IMAGE_CHECKSUM=http://media.suse.baremetal/ubuntu_22.04-efi.qcow2.md5
+export IMAGE_CHECKSUM_TYPE=md5
+export IMAGE_FORMAT=qcow2
+
+
+########################################
+# Kubeadm Control Plane configurations #
+########################################
+
+#
+# Network configuration. The NIC names must match the ones that were
+# discovered during baremetal host inspection. To see the NIC names
+# of a given baremetal host. Run "kubectl get bmh <baremetal host> -o yaml".
+# The name of the IP pool are coming from the clusterctl template.
+#
+# NOTE: these are YAML snippets to be injected into kubeadmConfigSpec so
+# make sure we keep the 4 spaces indentation for each line.
+#
+export CONTROL_PLANE_NETWORK_DATA="
+    links:
+      ethernets:
+      - id: ens1f0
+        macAddress:
+          fromHostInterface: ens1f0
+        type: phy
+      vlans:
+      - id: ens1f0.1076
+        vlanID: 1076
+        vlanLink: ens1f0
+        macAddress:
+          string: ec:fd:aa:0d:22:5f
+    networks:
+      ipv4:
+      - id: ens1f0
+        ipAddressFromIPPool: provisioning-pool
+        link: ens1f0
+        routes:
+        - gateway:
+            string: 192.168.75.254
+          network: 192.168.10.0
+          prefix: 21
+      - id: ens1f0.1076
+        link: ens1f0.1076
+        ipAddressFromIPPool: baremetalv4-pool
+        routes:
+        - gateway:
+            fromIPPool: baremetalv4-pool
+          network: 0.0.0.0
+    services:
+      dns:
+      - 192.168.75.7
+"
+
+#
+# User accounts for SSH into the workload cluster, for dev purposes.
+# If you don't want to create any user accounts, just set the var to
+# an empty string.
+#
+# NOTE: these are YAML snippets to be injected into kubeadmConfigSpec so
+# make sure we keep the 4 spaces indentation for each line.
+#
+export KUBEADM_CONTROL_PLANE_SSH_USERS="
+    users:
+    - name: metal3
+      shell: /bin/bash
+      sshAuthorizedKeys:
+      - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAICXtmbErb2jcl7iVZToyq+IRSgILaPC0V99C1pSsx83S
```

Comment:
I need to replace this with my personal key right?

Reply:
Yes